src/app/datapoints-graph/datapoints-graph-view/datapoints-graph-widget-view.component.html

```diff
@@ -82,62 +88,174 @@
+        <div
+          class="c8y-datapoint-pill flex-no-shrink"
+          *ngFor="let datapoint of displayConfig.datapoints"
+          [ngClass]="{ active: datapoint.__active }"
+          title="{{ datapoint.label }} - {{ datapoint.__target.name }}"
         >
-          <span class="text-truncate">
-            {{ datapoint.label }}
-          </span>
-          <small class="text-muted text-10">
-            {{ datapoint.__target.name }}
-          </small>
-        </span>
-        <i
-          *ngIf="datapointsOutOfSync.get(datapoint)"
-          [tooltip]="
-            'Measurements received for this data point may be out of sync.'
-              | translate
-          "
-          container="body"
-          [adaptivePosition]="false"
-          c8yIcon="exclamation-triangle"
-          class="text-warning m-l-4"
-        ></i>
+          <button
+            type="button"
+            title="{{
+              (datapoint.__active ? hideDatapointLabel : showDatapointLabel)
+                | translate
+            }} "
+            class="c8y-datapoint-pill__btn"
+            (click)="toggleChart(datapoint)"
+          >
+            <i
+              [c8yIcon]="
+                datapoint.__active ? 'eye text-primary' : 'eye-slash text-muted'
+              "
+              class="icon-14"
+            ></i>
+          </button>
+          <div class="c8y-datapoint-pill__label c8y-datapoint-pill__btn">
+            <i
+              c8yIcon="circle"
+              class="m-r-4 icon-14"
+              [ngStyle]="{
+                color: datapoint.color,
+              }"
+            ></i>
+            <span
+              class="text-truncate"
+              [ngClass]="{ 'text-muted': !datapoint.__active }"
+            >
+              <span class="text-truncate">
+                {{ datapoint.label }}
+              </span>
+              <small class="text-muted text-10">
+                {{ datapoint.__target.name }}
+              </small>
+            </span>
+            <i
+              *ngIf="datapointsOutOfSync.get(datapoint)"
+              [tooltip]="
+                'Measurements received for this data point may be out of sync.'
+                  | translate
+              "
+              container="body"
+              [adaptivePosition]="false"
+              c8yIcon="exclamation-triangle"
+              class="text-warning m-l-4"
+            ></i>
+          </div>
+        </div>
+        <!-- Alarms  -->
+
+        <div
+          class="c8y-alarm-pill flex-no-shrink"
+          *ngFor="let alarm of alarms"
+          title="{{ alarm.filters.type }} "
+        >
+          <i
+            *ngIf="
+              displayConfig?.activeAlarmTypesOutOfRange?.includes(
+                alarm.filters.type
+              )
+            "
+            [tooltip]="
+              'Alarm of this type is currently active and outside of the selected time range'
+                | translate
+            "
```

Comment:
```suggestion
            [tooltip]="
             {{ 'Alarm of this type is currently active and outside of the selected time range'
                | translate}}
            "
```

Reply:
[] brackets are taking care of evaluating the value. syntax `[tooltip]="'Some string' | translate"` is widely used in c8y ui repo. 
This one would also work
`tooltip="
             {{ 'Alarm of this type is currently active and outside of the selected time range'
                | translate}}
            "`
            but using both braces is redundant.